botfront/imports/api/graphql/index.js

```diff
@@ -11,6 +11,9 @@ import configResolver from './config/configResolver';
 import configTypes from './config/config.types.graphql';
 import nluResolvers from './nlu/resolvers';
 import nluTypes from './nlu/schemas';
+import trackerStoreResolver from './trackerStore/resolvers/trackerStoreResolver';
+import trackerStoreTypes from './trackerStore/schemas';
```

Comment:
Since this one doesn't have so many types, you could also do like for commonTypes and configTypes, but that doesn't really matter.